projects/plugins/backup/src/php/class-jetpack-backup.php

```diff
@@ -234,6 +243,31 @@ public static function get_backup_capabilities() {
 		);
 	}
 
+	/**
+	 * Gets information about the currently promoted backup product.
+	 *
+	 * @return string|WP_Error A JSON object of the current backup product being promoted if the request was successful, or a WP_Error otherwise.
+	 */
+	public static function get_backup_product_info() {
```

Comment:
I wonder if we should highlight in the method and endpoint name that this is for a single, promoted product? Maybe we could even switch to a new v2 endpoint that would return only the promoted product from WordPress.com, so we could update the promoted product without having to release a new version of the plugin?